sabnzbd/newsunpack.py

```diff
@@ -1192,6 +1171,7 @@ def PAR_Verify(parfile, parfile_nzf, nzo, setname, joinables, classic=False, sin
     stup, need_shell, command, creationflags = build_command(command)
 
     # par2 wants to see \\.\ paths on Windows
+    # See: https://github.com/sabnzbd/sabnzbd/pull/771
```

Comment:
`This kind of links might later work no more` 
yes, we should just make a hidden wiki page for dev notes? and then just link to the dev note  ID there instead?